Examples/Tests/ElectrostaticDirichletBC/PICMI_inputs_2d.py

```diff
@@ -0,0 +1,42 @@
+from pywarpx import picmi
```

Comment:
Adding this additional picmi test file is a good idea. However, can it be structured like the other files, defining the parameters first and then calling the picmi setup? A good example is `PICMI_inputs_laser_acceleration.py`.

Reply:
No problem, happy to change it accordingly. On a different note, this test keeps failing during the CI test, but locally it runs fine. Is there a way I can see the actual test report from Azure to see why it fails?